src/graphql/types/object/transaction.ts

```diff
@@ -60,7 +60,13 @@ const Transaction = GT.Object<WalletTransaction>({
         try {
           result = await loaders.txnMetadata.load(source.id)
         } catch (err) {
-          result = err
+          if (err instanceof Error) {
+            result = err
+          } else if (typeof err === "undefined") {
+            result = undefined
```

Comment:
do we need this branch?

Reply:
ah yeah probably not going to catch an undefined error